test-data/golden/chat_test_msg_with_implicit_member_add

```diff
@@ -0,0 +1,8 @@
+Group#Chat#10: Group chat [3 member(s)] 
+--------------------------------------------------------------------------------
+Msg#10:  (Contact#Contact#11): I created a group [FRESH]
+Msg#11:  (Contact#Contact#11): Member Fiona (fiona@example.net) added by alice@example.org. [FRESH][INFO]
+Msg#12: Me (Contact#Contact#Self): You removed member Fiona (fiona@example.net). [INFO] o
+Msg#13:  (Contact#Contact#11): Member Fiona (fiona@example.net) added. [FRESH]
```

Comment:
Yes, but the logic would be too complex. It can be a reply to old message, but it still can bring new membership changes if Alice has made them. This can really happen if Alice and Bob do it in parallel. The algo i suggested in https://github.com/deltachat/deltachat-core-rust/pull/4841#issuecomment-1776543605 solves this in a relatively simple way, one who did the changes later would win the game. But this is off-topic